release.sh

```diff
@@ -57,4 +57,4 @@ do
     wait $PID
 done
 
-cd ..
+cd ..
```

Comment:
Adding a newline at the of files is good practice. However, this change may not be relevant to the proposed feature request.

Reply:
Compressing with `gzip` adds one more dependency. Is `gzip` preinstalled on Arch?

Reply:
> @trueNAHO 
This newline is added by mistake . sorry I didn't notice. Its probably code formatter thing. :face_exhaling:

Reply:
@ful1e5 
'gzip' is probably not installed by default. I'm currently using Debian12 so I cann't confirm that. With `gzip` (which you are currently using) (`tar -czvf` )  the release file is ~8mb but if you use `tar -cJvf ` this command it will reduce its size to ~3mb.